README.md

```diff
@@ -19,18 +19,13 @@ This script guides you through deploying and testing the Reactive Faucet demo on
 * `SEPOLIA_RPC`
 * `SEPOLIA_PRIVATE_KEY`
 * `REACTIVE_RPC`
-* `REACTIVE_PRIVATE_KEY`
-* `DEPLOYER_ADDR`
+* `REACTIVE_PRIVATE_KEY
 
-[//]: # (* `CALLBACK_PROXY_ADDR`)
-
-[//]: # ()
-[//]: # (`DEPLOYER_ADDR` is your RVM ID. `CALLBACK_PROXY_ADDR` is a fixed EOA address, specific to each network, used by the Reactive Network to generate transaction callbacks.)
 You can use the recommended Sepolia RPC URL: `https://rpc2.sepolia.org`.
 
 ### Step 1
 
-Deploy the `ReactiveFaucetL1` contract to Sepolia and assign the `Deployed to` address from the response to `REACTIVE_FAUCET_L1_ADDR`.
+Deploy the `ReactiveFaucetL1` contract to Sepolia and assign the `Deployed to` address from the response to `REACTIVE_FAUCET_L1_ADDR`. Alternatively, you can export the pre-deployed `REACTIVE_FAUCET_L1_ADDR` on Sepolia, which is `0x9b9BB25f1A81078C544C829c5EB7822d747Cf434`.
 
 ```bash
 forge create --rpc-url $SEPOLIA_RPC --private-key $SEPOLIA_PRIVATE_KEY src/faucet/ReactiveFaucetL1.sol:ReactiveFaucetL1 --constructor-args 1ether
```

Comment:
This isn't right, the faucet deployment has changed too to accommodate single-line deployment. Check out the commit 41d40e0248b60da9580207b1979730f954405734 in our gitlab repo.

Reply:
Do you mean the following part should be removed: Alternatively, you can export the pre-deployed `REACTIVE_FAUCET_L1_ADDR` on Sepolia, which is `0x9b9BB25f1A81078C544C829c5EB7822d747Cf434`.

Reply:
My commands match yours on Gitlab

Reply:
why do we assign the contract address in Step 2 to REACTIVE_FAUCET_ADDR if we never use it later? 
we only use REACTIVE_FAUCET_L1_ADDR in step 3

Reply:
My mistake, I saw no change to the deployment command, and assumed you didn't add the `--value` option to it, not noticing it was done before that.

> why do we assign the contract address in Step 2 to REACTIVE_FAUCET_ADDR if we never use it later?

Just in case we want to pause/resume the faucet later, OR provide additional funds to it.